ticdc/integrate-confluent-using-ticdc.md

```diff
@@ -1,130 +1,264 @@
-        v TEXT
-    );
+    SELECT * FROM ORDERS EMIT CHANGES;
     ```
 
-    > **Note:**
-    >
-    > If you need to change the database name or the table name, change `topics` in `jdbc-sink-connector.json` accordingly.
+    ![Select from orders](/media/integrate/select-from-orders.png)
 
-2. Insert data into TiDB:
+    You can see that the incremental data has been replicated to ksqlDB, as shown in the preceding figure. Data integration with ksqlDB is done.
 
-    {{< copyable "sql" >}}
+## Integrate data with Snowflake
 
-    ```sql
-    INSERT INTO test (id, v) values (1, 'a');
-    INSERT INTO test (id, v) values (2, 'b');
-    INSERT INTO test (id, v) values (3, 'c');
-    INSERT INTO test (id, v) values (4, 'd');
-    ```
+Snowflake is a cloud native data warehouse. With Confluent, you can replicate TiDB incremental data to Snowflake by creating Snowflake Sink Connectors.
+
+### Prerequisites
+
+- You have registered and created a Snowflake cluster. See [Getting Started with Snowflake](https://docs.snowflake.com/en/user-guide-getting-started.html).
+- Before connecting to the Snowflake cluster, you have generated a private key for it. See [Key Pair Authentication & Key Pair Rotation](https://docs.snowflake.com/en/user-guide/key-pair-auth.html).
+
+### Integration procedure
+
+1. Create a database and a schema in Snowflake.
+
+    In the Snowflake control console, choose **Data** > **Database**. Create a database named `TPCC` and a schema named `TiCDC`.
+
+2. In the Confluent Cloud Console, choose **Data Integration** > **Connectors** > **Snowflake Sink**. The page shown below is displayed.
+
+    ![Add snowflake sink connector](/media/integrate/add-snowflake-sink-connector.png)
+
+3. Select the topic you want to replicate to Snowflake. Then go to the next page.
 
-3. Wait a moment for data to be replicated to the downstream. Then check the downstream for data:
+    ![Configuration](/media/integrate/configuration.png)
 
-    {{< copyable "shell-regular" >}}
+4. Specify the authentication information for connecting Snowflake. Fill in **Database name** and **Schema name** with the values you created in the previous step. Then go to the next page.
+
+    ![Configuration](/media/integrate/configuration.png)
+
+5. On the **Configuration** page, select `AVRO` for both **Input Kafka record value format** and **Input Kafka record key format**. Then click **Continue**. Wait until the connector is created and the status becomes **Running**, which might take several minutes.
+
+    ![Data preview](/media/integrate/data-preview.png)
+
+6. In the Snowflake console, choose **Data** > **Database** > **TPCC** > **TiCDC**. You can see that TiDB incremental data has been replicated to Snowflake. Data integration with Snowflake is done.
+
+## Integrate data with SQL Server
+
+SQL server is an RDBMS provided by Microsoft. With Confluent, you can replicate TiDB incremental data to SQL server by creating SQL Server Sink Connectors.
```

Comment:
an RDBMS or a RDBMS?

Reply:
https://forums.mysql.com/read.php?10,430972,430972#:~:text=short%20definition%20of-,an%20RDBMS,-is%3A%20a%20DBMS